tests/unit/preprocessor/_area/test_area.py

```diff
@@ -385,6 +420,18 @@ def test_crop_cube(make_testcube, square_shape, tmp_path):
         np.testing.assert_array_equal(result.data, expected)
 
 
+def test_crop_cube_with_ne_file():
+    """Test for cropping a cube by shape bounds."""
+    shp_file = "esmvalcore/preprocessor/ne_masks/ne_10m_ocean.shp"
```

Comment:
This shapefile appears to be used nowhere except in this test, why is it in the repository? And why do we use higher resolution shapefiles for masking land than sea?

Reply:
the `10m` is perfectly usable for oceans but it's overkill and very heavy on the speed, and the `50m` is good enough and much faster, we can probably remove the `10m` one and totally forget about it

Reply:
Sounds good, let's remove it. We can add it again if we ever need it.